crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/controller/StatelessMeansAssessmentController.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.laa.crime.meansassessment.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import jakarta.validation.Valid;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.justice.laa.crime.meansassessment.model.common.stateless.InitialMeansAssessment;
+import uk.gov.justice.laa.crime.meansassessment.model.common.stateless.StatelessApiRequest;
+import uk.gov.justice.laa.crime.meansassessment.model.common.stateless.StatelessApiResponse;
+import uk.gov.justice.laa.crime.meansassessment.model.common.stateless.Summary;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.StatelessOutcome;
+
+import java.math.BigDecimal;
+
+@RestController
+@RequestMapping("api/internal/v1/assessment/means/stateless")
```

Comment:
Not sure I like this, how about we just change the version part? I think this makes sense since MAAT will also use this endpoint in the future. `api/internal/v2/assessment/means/`